crates/rpc-types-trace/src/geth/call.rs

```diff
@@ -56,6 +56,9 @@ pub struct CallLogFrame {
     /// The data of the log.
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub data: Option<Bytes>,
+    /// The position of the log relative to subcalls within the same trace.
+    #[serde(default, with = "alloy_serde::quantity::opt")]
+    pub position: Option<u64>,
```

Comment:
should we skip serializing this if `None`?

Reply:
we should because null is generally invalid for this type of stuff

Reply:
Thanks for it, so we should also add skip_if_none